src/install.rs

```diff
@@ -461,7 +461,7 @@ async fn prepare_build<'a>(
     let bases = Bases::from_iter(actions.iter_build_pkgs().map(|p| p.pkg.clone()));
     let srcinfos = download_pkgbuilds(config, &bases).await?;
 
-    if !config.skip_review {
+    if config.skip_review == "never" {
```

Comment:
Shouldn't this be `!= always`?

Reply:
Yes, that it is true, good catch :) Fixed it.